packages/datagateway-common/src/api/datafiles.tsx

```diff
@@ -251,13 +338,26 @@ export const downloadDatafile = (
   };
 
   const link = document.createElement('a');
-  link.href = `${idsUrl}/getData?${Object.entries(params)
-    .map(([key, value]) => `${key}=${value}`)
-    .join('&')}`;
+  const objectUrl = content && window.URL.createObjectURL(content);
```

Comment:
I'm still wary of using `Blob`s for downloads - have we tested this new method when trying to download large files? e.g. can the 8 GB file here be downloaded via this method? https://datagateway-dev.isis.stfc.ac.uk/browse/instrument/57054468/facilityCycle/84793030/investigation/89234877/dataset/89234894/datafile?sort=%7B%22fileSize%22%3A%22desc%22%7D - via the current method I'm able to download this file (albeit it takes a while!)

Reply:
It's a bit different for a user to ask to preview a large file - where they'd probably expect it would be slow and impracticable to do so, than a user downloading it - where it happens in the browser's background so they're less affected by the wait

Reply:
Ah wait nevermind - I see that this is only used when the `content` param is supplied - which is only done in the previewer

Reply:
Yep, normal downloads won't use Blobs, it is only used for when the previewer has already downloaded the datafile as a Blob, and `downloadDatafile` acts more like save this Blob as a file to the computer.